service/history/history_engine.go

```diff
@@ -838,6 +842,16 @@ func (e *historyEngineImpl) NotifyNewHistoryEvent(
 	e.eventNotifier.NotifyNewHistoryEvent(notification)
 }
 
+func (e *historyEngineImpl) ChasmEngine() chasm.Engine {
```

Comment:
Is this called anywhere? Maybe I missed it?

Reply:
Thanks, removed